docs/components/multiselect.md

```diff
@@ -0,0 +1,719 @@
+# Multiselect
+
+<div v-if="hasMounted">
+
+**Multiselect** - Multiselect component
+
+- The total count of selected items will be displayed as the placeholder text when the multiselect is collapsed.
+- When items are selected they will be shown as dismissible badges above the filter input when the multiselect has focus.
+- If at least one item is selected, there will be a clear all button displayed in place of the dropdown chevron when the multiselect has focus.
+- Selected items are displayed at the top of the dropdown for easy access (refreshed on multiselect collapse).
+
+<div>
+  <KMultiselect label="Pick Something" :items="deepClone(defaultItemsUnselect)" />
+</div>
+
+```html
+<KMultiselect label="Pick Something" :items="items" />
+```
+
+## Props
+
+### items
+
+An array of items containing a `label` and `value`. You may also specify that a certain items are `selected` by default.
+
+<div>
+  <KMultiselect :items="deepClone(defaultItems)" />
+</div>
+
+```html
+<KMultiselect :items="[{
+    label: 'Cats',
+    value: 'cats',
+    selected: true
+  }, {
+    label: 'Dogs',
+    value: 'dogs'
+  }, {
+    label: 'Bunnies',
+    value: 'bunnies',
+    selected: true
+  },
+  {
+    label: 'Lions',
+    value: 'lions'
+  }, {
+    label: 'Tigers',
+    value: 'tigers'
+  }, {
+    label: 'Bears',
+    value: 'bears'
+  }, {
+    label: 'A long & truncated item',
+    value: 'long'
+  }]"
+/>
+```
+
+### label
+
+The label for the select.
+
+<div>
+  <KMultiselect label="Cool label" :items="deepClone(defaultItemsUnselect)" />
+</div>
+
+```html
+<KMultiselect label="Cool label" :items="items" />
+```
+
+### labelAttributes
+
+Use the `labelAttributes` prop to configure the **KLabel's** [props](/components/label.html) if using the `label` prop.
+
+<div>
+  <KMultiselect label="Name" :label-attributes="{
+      help: 'I use the KLabel `help` prop',
+      'data-testid': 'test'
+    }"
+    :items="defaultItemsUnselect"
+  />
+</div>
+
+```html
+<KMultiselect
+  label="Name"
+  :label-attributes="{
+    help: 'I use the KLabel `help` prop',
+    'data-testid': 'test'
```

Comment:
It seems setting `data-testid` has no effect. Maybe we should move `v-bind` below `data-testid` [here](https://github.com/Kong/kongponents/blob/da50fc4ab70a063cfb866c62eaccc058dbc267ab/src/components/KMultiselect/KMultiselect.vue#L10-L11).

Reply:
Oh, good catch! Fixed this.